### PR TITLE
Create STANDALONE_VIEWER env var

### LINKED ISSUE #2014: Create an environment variable gating off Standalone features
## Action Requested
Now that we've built out most standalone features we need to gate them behind an environment variable so we don't have standalone features popping up for folks who just want the viewer with NBS

## Acceptance Criteria

- [x] Create an env var: 
- [x] Gate ALL standalone viewer features behind this. Those listed below and anything you can think of that I missed
- [x] Saving to/retrieving from the  Metadata table
- [x] Ability to access to the homepage with the eCR table

## Testing
We will want some simple testing ensuring that standlone features aren't available without 

## Designs
Negative

## Background & Context
We need to draw a line between NBS and Standalone versions of the viewer so users dont end up with features they don't want!

### PR DESCRIPTION
**Pull request by lina-roth**
# PULL REQUEST

## Summary
- Add STANDALONE_VIEWER env var
- Block ability to access Standalone Viewer homepage if STANDALONE_VIEWER != true
- Block seeding of metadata  if STANDALONE_VIEWER != true
- Block retrieval of metadata  if STANDALONE_VIEWER != true

## Related Issue
Fixes #2014 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/seed-scripts/create-seed-data.py [hunk 1]
```diff
@@ -88,9 +88,24 @@ def convert_files():
                     print(f"Converted {folder} successfully.")
                 else:
                     print(f"Failed to convert {folder}. Response: {response.text}")
-    return fhir_bundles, metadata
+    if (
```
Comment (nickbristow) [inline containers/ecr-viewer/seed-scripts/create-seed-data.py:91]: Does this need to be two lines?
Can it just be ```if os.environ.get("STANDALONE_VIEWER") == "true"```

File: containers/ecr-viewer/src/app/api/services/listEcrDataService.ts [hunk 2]
```diff
@@ -51,7 +51,12 @@ export async function listEcrData(): Promise<Ecr[]> {
   if (process.env.SOURCE === S3_SOURCE) {
     const s3Data = await list_s3();
     const processedData = processListS3(s3Data);
-    await getFhirMetadata(processedData);
+    if (
+      process.env.STANDALONE_VIEWER &&
```
Comment (nickbristow) [inline containers/ecr-viewer/src/app/api/services/listEcrDataService.ts:55]: I think this only needs one line

File: containers/ecr-viewer/src/app/api/services/listEcrDataService.ts [hunk 3]
```diff
@@ -154,13 +173,21 @@ export const processListS3 = (
...
-  const data = await database.manyOrNone<EcrMetadataModel>(fhirMetadataQuery, [
-    ecrIds,
-  ]);
-  console.log(data);
-  return data;
+    const data = await database.manyOrNone<EcrMetadataModel>(
+      fhirMetadataQuery,
+      [ecrIds],
+    );
+    console.log(data);
```
Comment (nickbristow) [inline containers/ecr-viewer/src/app/api/services/listEcrDataService.ts:188]: remove

File: containers/ecr-viewer/src/app/page.tsx [hunk 4]
```diff
@@ -9,11 +9,24 @@ export const dynamic = "force-dynamic";
  * @returns The home page JSX component.
  */
 const HomePage: React.FC = async () => {
-  const listFhirData = await listEcrData();
+  let listFhirData: Ecr[] = [];
+  if (
+    process.env.STANDALONE_VIEWER &&
```
Comment (nickbristow) [inline containers/ecr-viewer/src/app/page.tsx:14]: i think we only need the === "true" part

**Review by nickbristow — CHANGES_REQUESTED** [review]

File: containers/ecr-viewer/src/app/api/services/listEcrDataService.ts (hunk 3 above)
Comment (lina-roth) [reply containers/ecr-viewer/src/app/api/services/listEcrDataService.ts:188]: This was a change Boban added - see https://github.com/CDCgov/phdi/pull/2016/files#diff-735755c2a016620033148d756b9e8a9b80cbd4e7f28d0741f6606d17cec56778R164

I believe as part of the story it said to log the metadata

File: containers/ecr-viewer/src/app/api/services/listEcrDataService.ts (hunk 2 above)
Comment (lina-roth) [reply containers/ecr-viewer/src/app/api/services/listEcrDataService.ts]: You are correct. Updated

File: containers/ecr-viewer/src/app/page.tsx (hunk 4 above)
Comment (lina-roth) [reply containers/ecr-viewer/src/app/page.tsx]: You are correct. Updated

File: containers/ecr-viewer/seed-scripts/create-seed-data.py (hunk 1 above)
Comment (lina-roth) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py]: You are correct. Updated

File: containers/ecr-viewer/src/app/api/services/listEcrDataService.ts (hunk 3 above)
Comment (nickbristow) [reply containers/ecr-viewer/src/app/api/services/listEcrDataService.ts:176]: 👍

**Review by nickbristow — APPROVED** [review]

**Pull request merged**